### PR TITLE
Fix input prompt in hosting:channels:create

### PR DESCRIPTION
**Pull request by sarajmunjal**
Fixes issue: https://github.com/firebase/firebase-tools/issues/2719

### CODE REVIEW

File: src/commands/hosting-channel-create.ts [hunk 1]
```diff
@@ -45,11 +45,13 @@ export default new Command("hosting:channel:create [channelId]")
...
-        channelId = await promptOnce({
-          type: "input",
-          message: "Please provide a URL-friendly name for the channel:",
-          validate: (s) => s, // Prevents an empty string from being submitted!
-        });
+        channelId = await promptOnce(
+          {
+            type: "input",
+            message: "Please provide a URL-friendly name for the channel:",
+            validate: (s) => s.length > 0,
```
Comment (bkendall) [inline src/commands/hosting-channel-create.ts:52]:
```suggestion
            validate: (s) => s.length,
```

Comment (bkendall) [reply src/commands/hosting-channel-create.ts:52]: A suggestion

**Review by bkendall — APPROVED** [review]
Can you add a `CHANGELOG.md` entry?

```markdown
- Fixes input issue when prompting for a Hosting Channel name.
```

File: src/commands/hosting-channel-create.ts (hunk 1 above)
Comment (sarajmunjal) [reply src/commands/hosting-channel-create.ts:52]: Letting this be, because it gives me a lint error in vscode otherwise.

**Pull request merged**